server/model/monitor.js

```diff
@@ -253,6 +267,36 @@ class Monitor extends BeanModel {
                 if (await Monitor.isUnderMaintenance(this.id)) {
                     bean.msg = "Monitor under maintenance";
                     bean.status = MAINTENANCE;
+                } else if (this.type === "group") {
+                    const children = await Monitor.getChildren(this.id);
+
+                    if (children.length > 0) {
+                        bean.status = UP;
+                        bean.msg = "All childs up and running";
```

Comment:
Just me being a bit nitpicky should this be "children"? I think it just reads better.
```suggestion
                        bean.msg = "All children up and running";
```